.changeset/violet-brooms-warn.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/admin.login-flow.ai.v1": minor
+"@wso2is/admin.branding.ai.v1": minor
+---
+
+Add LoginFlow AI and BrandingAI to the Identity Server
```

Comment:
```suggestion
Add LoginFlow AI and Branding AI to the Identity Server
```

Reply:
This is fixed with https://github.com/wso2/identity-apps/pull/7391/commits/501827bfce53bd4964d2c10dd035d056e556aa55